gems/pending/appliance_console/internal_database_configuration.rb

```diff
@@ -1,23 +1,24 @@
 require "appliance_console/database_configuration"
 require "appliance_console/service_group"
+require "util/postgres_admin"
 
 module ApplianceConsole
   class InternalDatabaseConfiguration < DatabaseConfiguration
-    DATABASE_DISK_FILESYSTEM_TYPE = "xfs".freeze
-    POSTGRES_USER                 = "postgres".freeze
-    POSTGRES_DIR                  = "opt/rh/postgresql92/root/var/lib/pgsql/data".freeze
-    DATABASE_DISK_MOUNT_POINT     = Pathname.new("/").join(POSTGRES_DIR).freeze
-    VOLUME_GROUP_NAME             = "vg_data".freeze
-    LOGICAL_VOLUME_NAME           = "lv_pg".freeze
-    LOGICAL_VOLUME_PATH           = Pathname.new("/dev").join(VOLUME_GROUP_NAME, LOGICAL_VOLUME_NAME).freeze
-    SCL_ENABLE_PREFIX             = "scl enable postgresql92".freeze
-    CERT_LOCATION                 = Pathname.new("/var/www/miq/vmdb/certs").freeze
-    POSTGRESQL_SAMPLE             = Pathname.new("/var/www/miq/system/COPY/").join(POSTGRES_DIR).freeze
-    POSTGRESQL_TEMPLATE           = Pathname.new("/var/www/miq/system/TEMPLATE/").join(POSTGRES_DIR).freeze
-    POSTGRESQL_SERVICE            = "postgresql92-postgresql".freeze
     attr_accessor :disk
     attr_accessor :ssl
 
+    def self.postgres_dir
+      PostgresAdmin.data_directory.relative_path_from(Pathname.new("/"))
+    end
+
+    def self.postgresql_sample
+      Rails.root.join("../system/COPY").join(postgres_dir)
```

Comment:
@matthewd The problem is that the system/COPY,LINK directories and the mechanism that copies/symlinks to those locations is dumb.  We blindly copy/link like a chroot below those directories and don't look at what we're copying/linking.  Therefore, the [well-known path](https://github.com/ManageIQ/manageiq-appliance/pull/9/files) is also changing for this work.  The value directory structure below the COPY directory is directly related to the PostgresAdmin.data_directory.

So, we either teach the dumb copier/linker to copy/link to the PostgresAdmin.data_directory or we duplicate this logic by moving files around when we upgrade postgresql.  I went with that I thought was easier to maintain.